corehq/apps/app_manager/views.py

```diff
@@ -335,6 +336,22 @@ def copy_app(request, domain):
 
 
 @require_can_edit_apps
+def app_from_template(request, domain, slug):
+    _clear_app_cache(request, domain)
+    template = load_app_template(slug)
+    app = import_app_util(template, domain, {
+        'created_from': 'template_%s' % slug,
```

Comment:
Is the `template_` prefix necessary? If we need to do anything with this at a later point in time we'll have to strip it off again.